src/bodyScrollLock.js

```diff
@@ -1,10 +1,4 @@
-// @flow
-import UAParser from 'ua-parser-js';
-// Adopted and modified solution from Bohdan Didukh (2017)
-// https://stackoverflow.com/questions/41594997/ios-10-safari-prevent-scrolling-behind-a-fixed-overlay-and-maintain-scroll-posi
-
```

Comment:
Prefer to retain comments from lines 3 to 5 to continue giving credit where it is due